src/main/java/com/softserveinc/Task03.java

```diff
@@ -1,11 +1,33 @@
 package com.softserveinc;
 
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class Task03 {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
-        // TODO: Write your code here
+        System.out.print("Write how many ml of water the coffee machine has:\n" + ">>> ");
+            int CupsFromWater = scanner.nextInt() / 200; //
+        System.out.print("Write how many ml of milk the coffee machine has:\n" + ">>> ");
+            int CupsFromMilk = scanner.nextInt() / 50;
```

Comment:
But don't forget about code style. You can use keyboard shortcut `Ctrl-Alt-L` to format your code.